Documentation/helm-operator.md

```diff
@@ -95,59 +95,60 @@ The command removes all the Kubernetes components associated with the chart and
+| Parameter                          | Description                                                                                                                 | Default                                                |
+| ---------------------------------- | --------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------ |
+| `image.repository`                 | Image                                                                                                                       | `rook/ceph`                                            |
+| `image.tag`                        | Image tag                                                                                                                   | `master`                                               |
+| `image.pullPolicy`                 | Image pull policy                                                                                                           | `IfNotPresent`                                         |
+| `rbacEnable`                       | If true, create & use RBAC resources                                                                                        | `true`                                                 |
+| `pspEnable`                        | If true, create & use PSP resources                                                                                         | `true`                                                 |
+| `resources`                        | Pod resource requests & limits                                                                                              | `{}`                                                   |
+| `annotations`                      | Pod annotations                                                                                                             | `{}`                                                   |
+| `logLevel`                         | Global log level                                                                                                            | `INFO`                                                 |
+| `nodeSelector`                     | Kubernetes `nodeSelector` to add to the Deployment.                                                                         | <none>                                                 |
+| `tolerations`                      | List of Kubernetes `tolerations` to add to the Deployment.                                                                  | `[]`                                                   |
+| `unreachableNodeTolerationSeconds` | Delay to use in node.kubernetes.io/unreachable toleration                                                                   | `5`                                                    |
```

Comment:
suggestions: 
- "Delay to use for the node.kubernetes.io/unreachable pod failure toleration to override the Kubernetes default of 5 minutes"
- For the default: `5s` instead of `5`